src/Tribe/Aggregator/Errors.php

```diff
@@ -56,5 +56,7 @@ private function __construct() {
 		tribe_register_error( 'core:aggregator:record-not-finalized', __( 'Import records must be finalized before posts can be inserted.', 'the-events-calendar' ) );
 		tribe_register_error( 'core:aggregator:save-child-failed', __( 'Unable to save scheduled import instance. Please try again.', 'the-events-calendar' ) );
 		tribe_register_error( 'core:aggregator:save-schedule-failed', __( 'Unable to save scheduled import. Please try again.', 'the-events-calendar' ) );
+
```

Comment:
No big deal, but why the extra line break - seems like the following line might belong in a block with the other tribe_register_error() calls?